policyengine_us/variables/gov/local/co/denver/dhs/property_tax_relief/eligibility/co_denver_property_tax_relief_renter_eligible.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class co_denver_property_tax_relief_renter_eligible(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Eligible for the renter Denver Property Tax Relief"
+    definition_period = YEAR
+    reference = "https://denvergov.org/files/content/public/v/37/government/agencies-departments-offices/agencies-departments-offices-directory/denver-human-services/be-supported/additional-assistance/property-tax-relief/denver-property-tax-relief-program-year-2021-rules.pdf"
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(period).gov.local.co.denver.dhs
+        pays_rent = add(spm_unit, period, ["rent"]) > 0
+        has_elderly_or_disabled = spm_unit(
+            "has_co_denver_dhs_elderly_disabled", period
+        )
+        renters_with_elderly_or_disabled = has_elderly_or_disabled & pays_rent
+
+        income = spm_unit("co_denver_property_tax_relief_income", period)
+        size = spm_unit("spm_unit_size", period)
+        ami = p.ami.calc(size)
```

Comment:
Actually please move this to the `ami` variable. It's defined for four-person households, then adjusted using flat constant ratios from there (please verify they align).

Reply:
I think our current approach is a better way to reflect the legal code, writing it in a four-person households as a base will result in a lot more ratio constants that are not defined in the legal code.